apigee.properties

```diff
@@ -2,11 +2,17 @@ category=External Analyzers
 description=Adds XML rules test Apigee apiproxies.
 homepageUrl=https://github.com/CreditMutuelArkea/sonar-apigee-plugin
 archivedVersions=1.1.0,1.2.0,1.3.0
-publicVersions=2.0.0
+publicVersions=2.1.0
 
 defaults.mavenGroupId=com.arkea.satd.sonar.apigee
 defaults.mavenArtifactId=sonar-apigee-plugin
 
+2.1.0.description=New features
+2.1.0.sqVersions=[6.7,LATEST]
```

Comment:
7.9 is the oldest version still supported in the Marketplace. Please start the range for this plugin version there.

Reply:
Problem here, my company still uses SQ 7.6.
Is it possible to make an exception and start the range at 7.6?

Reply:
This is not about policy, but about how the metadata generator that consumes these files works. However, regardless of the range here, you can install the plugin manually in your company's local instance without problems.

Reply:
I agree. (And we'll install manually). Fix is done.